src/plugins/gui/providers/banxaProvider.ts

```diff
@@ -105,7 +105,6 @@ const asBanxaTxLimit = asObject({
 })
 
 const asBanxaPaymentType = asValue(
-  'CHECKOUTCREDIT',
```

Comment:
misunderstanding of the task. The goal is to remove the old paymentType but replace it with the new equivalent. See the Banxa chat